pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -424,7 +424,7 @@ def tz_convert_from_utc(int64_t[:] vals, tzinfo tz):
     int64 ndarray of converted
     """
     cdef:
-        int64_t[:] converted
+        const int64_t[:] converted
```

Comment:
i dont think const is needed here

Reply:
This is a similar case as below, once we start returning a const slice we have to declare this const too :slightly_frowning_face: 

```
Error compiling Cython file:
------------------------------------------------------------
...
        int64_t[:] converted

    if len(vals) == 0:
        return np.array([], dtype=np.int64)

    converted = _tz_convert_from_utc(vals, tz)
                                   ^
------------------------------------------------------------

pandas/_libs/tslibs/tzconversion.pyx:432:36: Memoryview 'const int64_t[:]' not conformable to memoryview 'int64_t[:]'.
```

Reply:
ok, thanks for humoring me